scripts/run_test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+docker system prune --all --force
+make run-nsm-test-cpp
+./command-executer/command_executer_docker_dir/command-executer run --cid 16 --port 5005 --command "/nsm-test/test" --no-wait
```

Comment:
I think the `--no-wait` here is not correct if the script is supposed to get the status code from the executed program.